ami-variables-user-data.sh.example

```diff
@@ -1,5 +1,8 @@
 #!/bin/bash
 
+# Template:
+# echo "MY_VAR=value" >> /etc/sysconfig/log-export-container
+
 echo "LOG_EXPORT_CONTAINER_OUTPUT=mongo" >> /etc/sysconfig/log-export-container
```

Comment:
Change this 2 (by default) values to be:
* `LOG_EXPORT_CONTAINER_INPUT=syslog-json`
* `LOG_EXPORT_CONTAINER_OUTPUT=stdout`